src/bloom.h

```diff
@@ -115,9 +115,6 @@ class CBloomFilter
 class CRollingBloomFilter
 {
 public:
-    // A random bloom filter calls GetRand() at creation time.
-    // Don't create global CRollingBloomFilter objects, as they may be
-    // constructed before the randomizer is properly initialized.
```

Comment:
Is there a particular reason this comment was removed?

Reply:
@MarcoFalke says it doesn't apply anymore.

Reply:
Ah sorry, didn't see!